sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/BlobCheckpointStore/BlobsCheckpointStore.cs

```diff
@@ -344,7 +344,8 @@ await foreach (BlobItem blob in ContainerClient.GetBlobsAsync(traits: BlobTraits
 
             async Task<List<EventProcessorCheckpoint>> listLegacyCheckpointsAsync(List<EventProcessorCheckpoint> existingCheckpoints, CancellationToken listCheckpointsToken)
             {
-                var legacyPrefix = string.Format(CultureInfo.InvariantCulture, LegacyCheckpointPrefix, fullyQualifiedNamespace.ToLowerInvariant(), eventHubName.ToLowerInvariant(), consumerGroup.ToLowerInvariant());
+                // Legacy checkpoints are not normalized to lowercase
+                var legacyPrefix = string.Format(CultureInfo.InvariantCulture, LegacyCheckpointPrefix, fullyQualifiedNamespace, eventHubName, consumerGroup);
```

Comment:
Without this change, are the legacy checkpoints skipped? Do we have a test for this?

Reply:
That's an interesting question...  I think that Storage is case-sensitive for naming, but I'm not 100% sure.

Reply:
Yes, I changed the existing tests to use the expected casing both to T1 and T2 checkpoints. Storage is case-sensitive.